raiden/blockchain_events_handler.py

```diff
@@ -519,9 +519,9 @@ def handle_channel_batch_unlock(raiden: "RaidenService", event: Event):
     )
     assert token_network_state is not None
 
-    if participant1 == raiden.address:
+    if participant1 == raiden.address or participant1 in token_network_state.channelidentifiers_to_channels:
```

Comment:
i dont understand the `participant1 in token_network_state.channelidentifiers_to_channels:` part here

if the two light clientes are into the same hub, both are going to be present on  `token_network_state.channelidentifiers_to_channels`: right?

could you explain how this works?

Reply:
Yes. This codes just checks if the node/lc is a participant of the channel, what is needed in order to know if it must dispatch a state_change

Reply:
but suposse two light clients into the same hub, this will assign arbitrary..

Reply:
This participant is only used to fetch the channel state from db (something like what was done on unlock), and check its existence. So for this purpose is not relevant. 

Anyway, I agree its not very clean, but I didn't want to touch this part to much.